src/leiningen/ring/war.clj

```diff
@@ -132,13 +132,27 @@
     (generate-resolve handler-sym)))
 
 (defn compile-servlet [project]
-  (let [servlet-ns  (symbol (servlet-ns project))]
+  (let [servlet-ns  (symbol (servlet-ns project))
+        init-sym    (get-in project [:ring :servlet-init])
+        destroy-sym (get-in project [:ring :servlet-destroy])]
     (compile-form project servlet-ns
       `(do (ns ~servlet-ns
-             (:gen-class :extends javax.servlet.http.HttpServlet))
+             (:gen-class
```

Comment:
I don't think adding a line break is necessary here, as it still fits under 80 characters.

Reply:
It is by no means necessary, but there is more stuff coming after it now, and they don't align nicely w/o that newline. I will revert though...

Reply:
done